README.md

```diff
@@ -6,29 +6,59 @@ This consists of using AWS API Gateway with a Custom Domain to host the MTA-STS
 
 ## How to use the example code
 
-There are three examples for using the module in different ways.
+There are three examples for using the module in different ways:
 
-## Guided Automatic /examples/guided-automatic
+1. Guided Automatic using Route 53
+2. Manual mode using another DNS provider
+3. Manual mode using Route 53
+
```

Comment:
Add pre-requisites either here or in each section depending how specific you want to be. For example:

a live domain with p

- ublic DNS records
- GitHub on your local machine (desktop or CLI) - https://desktop.github.com/
- Terraform (https://www.terraform.io/downloads.html)
- Admin access to the AWS Route 53 where the DNS records are hosted